### DISCUSSION TITLE
The session is not replicated in the different domains

### DISCUSSION THREAD

**Discussion by aliriogonzalez** [jasny/sso#146] [Help needed]
Hello good evening. I have implemented Jasny SSO, my local scenario is as follows:

SSO Server: http://ssoserverb2b.com/
Broker 1: http://ssoclient1.com/
Broker 2: http://ssoclient2.com/

My problem is the following, when I log in to Broker 1 the session is not kept alive in Broker 2, that is, I have to log in to Broker 2.

Thank you very much in advance, and my apologies if I'm wrong on any point.
![Screenshot 2023-01-18 211517](https://user-images.githubusercontent.com/35611891/213339903-c8fe1845-ebd4-478d-bcca-3c6cc8b37387.png)
![Screenshot 2023-01-18 211719](https://user-images.githubusercontent.com/35611891/213339904-b10beef4-bb30-4c15-953f-4a8acf30bb83.png)

**Comment by jasny** [comment]
Please enable logging to find out what's going on.

Also, each domain should get it's own broker id and secret. The should not both be Alirio.

**Comment by espinozaguillermo** [comment]
@jasny Hi, I have the same problem implementing the library. 
I configured two brokers with different id and secret. The session in the server when attached the brokers are created but there aren't shared across the brokers.
Is there any special php session configuration ?

**Reply by jasny** [reply]
No. The only session that exist is the session on the server. The brokers can query that session on behalf of the user with the sso token. Each SSO token is linked to a session on the server.

Please read the wiki article to make sure you understand the concept, and the enable debugging to find the issue.
